extensions/resource-deployment/src/ui/notebookWizard/notebookWizardModel.ts

```diff
@@ -0,0 +1,11 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { Model } from '../model';
+
+export class NotebookWizardModel extends Model {
```

Comment:
is this really needed?

Reply:
Yeah can use Model directly but as a design pattern it is good to have its own Model type for the notebookWizard, perhaps  I can change it to type which in future could be changed to a class if needed.